src/plugins/intel_cpu/src/nodes/eltwise.cpp

```diff
@@ -697,6 +698,10 @@ struct jit_uni_eltwise_generic : public jit_uni_eltwise_kernel, public jit_gener
                     vpmovzxwd(vmm_src, op);
                     uni_vpslld(vmm_src, vmm_src, 16);
                     break;
+                case Precision::FP16:
+                    assert(mayiuse(x64::avx512_core_fp16));
```

Comment:
BTW, from my understanding conversion instructions like `vcvtph2ps` don't actually require `avx512_core_fp16` ISA, but just F16C + AVX512VL/AVX512F (or pure avx2)  which is available on all modern intel CPUs. Given that we can relax ISA limitation for all operations that uses only FP32<->FP16 conversion and keep real math in FP32 (like Eltwise, MVN, Interpolate etc). By doing that we can enable FP16 tests for such layers in precommit already now. 
What do you think? Sounds like worth to add in follow-up PR.

Reply:
OK, let me remove these assert and test can be added in follow-up PR

Reply:
BTW, one exception is `vcvtss2sh/vcvtsh2ss` used in  `load_scalar/store_scalar` which requires AVX512-FP16, Can we changed them into using `vcvtps2ph/vcvtph2ps` instead?

Reply:
For Eltwise it is save. 
Not sure about load/store_emitters. Would ask @chenhu-wang to comment.

Reply:
OK, that's enough, load/store_emitters actually already using vector version `vcvtps2ph/vcvtph2ps` with mask to handle variable length load/store

Reply:
You are still using `avx512_core_fp16` check which, from my understanding, is available on SPR only. In other words `avx512_core_fp16` is not equal `f16c + avx512f + avx512vl`. So to enable single layer tests in precommit we need to relax isa limitation.

Reply:
Yes,  indeed, some avx512_core_fp16 checks are still there, sorry I didn't remove them completely, and we can do that in single layer tests PR.

Reply:
Just a reminder that sse4 do not support f16<-> f32 convert instructions. Instead of report exceptions or assert, there should be some alignment, in property(precision hint) reading stage or createSupportedPrimitive() stage of nodes.